LocalWiki.php

```diff
@@ -200,6 +200,16 @@ function onClosedSiteNoticeAfter( &$siteNotice, $skin ) {
 	$wgGroupPermissions['*']['createtalk'] = false;
 }
 
+if ( $wgDBname === 'sau226wiki' ) {
+	$wgGroupPermissions['sysop']['abusefilter-modify-restricted'] = false;
+	$wgGroupPermissions['sysop']['editinterface'] = false;
+	$wgGroupPermissions['sysop']['globalblock-whitelist'] = false;
+	$wgGroupPermissions['sysop']['ipblock-exempt'] = false;
+	$wgGroupPermissions['sysop']['nuke'] = false;
+	$wgGroupPermissions['bureaucrat']['managewiki'] = false;
```

Comment:
Please do this with wmgManageWikiGroup rather than like this.

Reply:
That's fine by me. However you appear to use manual overrides for other wikis so I need clarification for this. I have dealt with managewiki by LocalSettings.php and have removed them from LocalWiki.php

Reply:
I have done it with the intention that if it doesn't work I will define manual overrides in LocalWiki.php if LocalSettings.php doesn't work